CMakeLists.txt

```diff
@@ -353,7 +353,7 @@ option_requires(VRPN_BUILD_PYTHON_HANDCODED_3X
 ###
 # javac, jar, and javah (for java wrapper)
 ###
-if(ANDROID)
+if(ANDROID AND DEFINED find_host_package)
```

Comment:
The build succeeds but does this mean the build doesn't see Java/JNI packages on Android? Also do you need a similar test for find_host_program?

Reply:
It doesn't matter, actually, since we don't use the VRPN JNI bindings on android, but I think actually the initial usage of find_host_package here is probably mistaken anyway.

No, no extra check is needed because the toolchain files that provide one provide both.